src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -172,6 +173,36 @@ def load_arguments(self, _):
         c.ignore('language')
         c.ignore('using_webapp_up')
 
+    with self.argument_context("webapp sitecontainers") as c:
+        c.argument('name', arg_type=webapp_name_arg_type, help='Name of the linux webapp')
+        c.argument("container_name", help='Name of the SiteContainer')
+        c.argument('slot', options_list=['--slot', '-s'], help='Name of the web app slot. Default to the productions slot if not specified.')
+
+    with self.argument_context("webapp sitecontainers create") as c:
+        c.argument("image", help='Image Name')
+        c.argument("target_port", help='Target port for SiteContainer')
+        c.argument("startup_cmd", help='Startup Command for the SiteContainer')
+        c.argument("is_main", help="true if the container is the main SiteContainer; false otherwise",
+                   arg_type=get_three_state_flag())
+        c.argument("system_assigned_identity", help="If true, the system-assigned identity will be used for auth while pulling image",
+                   arg_type=get_three_state_flag())
+        c.argument("user_assigned_identity", help='ClientID for the user-maganed identity which will be used for auth while pulling image')
+        c.argument("registry_username", help='Username used for image registry auth')
+        c.argument("registry_password", help='Password used for image registry auth')
+        c.argument("sitecontainers_spec_file", help="Path to a json sitecontainer spec file containing a list of sitecontainers, other sitecontainer input args will be ignored if this arg is provided")
+
+    with self.argument_context("webapp sitecontainers update") as c:
+        c.argument("image", help='Image Name')
+        c.argument("target_port", help='Target port for SiteContainer')
+        c.argument("startup_cmd", help='Startup Command for the SiteContainer')
+        c.argument("is_main", help="true if the container is the main site container; false otherwise",
+                   arg_type=get_three_state_flag())
+        c.argument("system_assigned_identity", help="If true, the system-assigned identity will be used for auth while pulling image",
+                   arg_type=get_three_state_flag())
+        c.argument("user_assigned_identity", help='ClientID for the user-maganed identity which will be used for auth while pulling image')
```

Comment:
please follow [this guideline](https://github.com/Azure/azure-cli/blob/20ffb2d35d59a104641d75aeb76b6f31885e26aa/doc/managed_identity_command_guideline.md) to develop managed identity related features.

Reply:
for this command, we are not enabling/disabling managed identity and thus we are not using `--system-assigned` or `--user-assgined` like other commands do

We have used different param name `--system-assigned-identity` or `--user-assigned-identity` and it is just an input to figure out which identity to use for pulling an image from user's container registry. We have also documented it accordingly.